packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -123,6 +123,31 @@ cluster.addAutoScalingGroup(autoScalingGroup);
 
 If you omit the property `vpc`, the construct will create a new VPC with two AZs.
 
+## Spot Instances
```

Comment:
I think this one should be placed as a subsection to the "Clusters" section.

Reply:
fixed https://github.com/aws/aws-cdk/pull/4360/commits/45bef1da7735bc55ff50668e70c7c3c9349b6944